Tests/NIOTests/BootstrapTest.swift

```diff
@@ -529,6 +529,23 @@ class BootstrapTest: XCTestCase {
         XCTAssertNil(NIOPipeBootstrap(validatingGroup: elg))
         XCTAssertNil(NIOPipeBootstrap(validatingGroup: el))
     }
+    
+    func testShorthandOptionsAreEquivalent() throws {
+        func bindAndGetReuseAddrOption(applyBootstrapOptions : (ServerBootstrap) -> ServerBootstrap) throws -> ChannelOptions.Types.SocketOption.Value {
+            let bootstrap = applyBootstrapOptions(ServerBootstrap(group: group))
+            let serverChannel = try bootstrap.bind(host: "127.0.0.1", port: 0).wait()
+            let reuseValue = try serverChannel.getOption(ChannelOptions.socketOption(.reuseaddr)).wait()
+            try serverChannel.close().wait()
+            return reuseValue
+        }
+        
+        let sbLongReuseValue = try bindAndGetReuseAddrOption(applyBootstrapOptions: { bs in bs.serverChannelOption(ChannelOptions.socketOption(.reuseaddr), value: 1) })
```

Comment:
consider reformatting to

```
let sbShortReuseValue = try bindAndGetReuseAddrOption { bs in
    bs.serverOptions([.reuseAddr])
}
```

usually, the trailing closure syntax is preferred (and if possible we should avoid lines that are longer than the github review window is wide (I think about 120).

Reply:
And whilst we're here, could we also compare against the traditional API?

    serverChannelOption(.socket(.init(SOL_SOCKET), .init(SO_REUSEADDR)))

Reply:
Did the reformat.
Traditional API I think is so traditional that no examples existed in the code - I believe what you're suggesting is equivalent to the 
let sbLongReuseValue = try bindAndGetReuseAddrOption { bs in
            bs.serverChannelOption(ChannelOptions.socketOption(.reuseaddr), value: 1) }
anyway?

Reply:
Done.

Reply:
OK, I think I've added the traditional API too.